ci-operator/config/openshift/kubernetes/openshift-kubernetes-master.yaml

```
@@ -238,10 +238,22 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-jenkins
-- as: e2e-upgrade
+- as: e2e-gcp-upgrade
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-loki
+- as: e2e-aws-upgrade
+  steps:
+    cluster_profile: aws
+    workflow: openshift-upgrade-aws-loki
+- as: e2e-azure-upgrade
+  steps:
+    cluster_profile: azure4
+    workflow: openshift-upgrade-azure-loki
+- as: e2e-aws-selfupgrade
+  steps:
+    cluster_profile: aws
+    workflow: openshift-selfupgrade-aws-loki
```

```suggestion
    workflow: openshift-selfupgrade-aws-loki
    env:
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:latest"
      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:initial"
```
You may not need a new workflow for that - try this?

---

done